indico/modules/events/papers/templates/reviewing/timeline.html

```diff
@@ -42,13 +57,24 @@
                 </div>
                 {% set submission_info_class = 'weak-hidden' if not revision.is_last_revision %}
                 <div class="i-box-content submission-info {{ submission_info_class }}">
-                    {{ _render_file_list(revision.files) }}
+                    {{ _render_paper_file_list(revision.files) }}
                 </div>
             </div>
         </div>
     </div>
 {% endmacro %}
 
+{% macro _render_spotlight_file(file) %}
+    {% set icon = icon_from_mimetype(file.content_type, 'icon-file-filled') %}
+    <div class="flexrow f-a-center spotlight-file i-button text-color outline dashed paper-file"
+         title="{{ file.filename }}">
+        <a class="attachment spotlight-file-icon {{ icon }}"></a>
```

Comment:
Why use another `<a>` for it? Can't we use a single element for the link and big icon?